app/models/user.rb

```diff
@@ -1,4 +1,40 @@
 class User < ApplicationRecord
+  # This module represents a deleted user.  In particular as it
+  # relates to rendering rich content in tags.  This "Null" object
+  # provides the interface for rendering.
+  #
+  # @see UserTag for implementation details.
+  module DeletedUser
```

Comment:
We already have a `Users` namespace, how would you feel about `Users::DeletedUser` instead? This class is already pretty long and if we ever introduce another one of these (like e.g. `LoggedOutUser`) I'd rather have that in its own file too instead of embedded in `User`.

Reply:
Thanks for the reminder on the `Users` namespace, I'm good moving it to `Users::DeletedUser`.  I'll work on that.